### DISCUSSION TITLE
Сворачивать в трей как у YouTube-Music

### DISCUSSION THREAD

**Discussion by verasminsk** [cucumber-sp/yandex-music-linux#120] [Ideas]
Как на счет добавить возможность сворачивать в трей как это реализовано у [youtube-music](https://github.com/th-ch/youtube-music)? =)

![image](https://github.com/user-attachments/assets/fcd98f64-7744-4fdf-9a80-a004c521abc1)

![image](https://github.com/user-attachments/assets/ab8dfc1c-5fd1-4a57-82db-ad9022bf16f9)

**Comment by verasminsk** [comment]
И по скролу мышки на иконке регулировать громкость )

![image](https://github.com/user-attachments/assets/8b7b90a2-745a-434c-9295-04922ead7f37)

**Comment by cucumber-sp** [comment]
Так трей уже есть. Прочитайте Readme раздел с конфигом. Про скролл интересное предложение

**Comment by verasminsk** [comment]
Спасибо за подсказку :)

- Mожно ко клику иконки в трее - play/pause ?

- A еще можно ли при закрытии/открытия приложения продолжить тот же трек?

- и самое интересное, будет ли скин как в [Yaamp](https://github.com/umnik1/yaamp) =]

**Reply by cucumber-sp** [reply]
> Спасибо за подсказку :)
> 
> * Mожно ко клику иконки в трее - play/pause ?
> * A еще можно ли при закрытии/открытия приложения продолжить тот же трек?
> * и самое интересное, будет ли скин как в [Yaamp](https://github.com/umnik1/yaamp) =]

Там и так проблемы с определением правого/левого клика во многих дистрибутивах. Так что нет. В системном плеере в гноме появляется, так что можно переключать там. Скинов и доп. функционала не планируется. Главная цель проекта - поддержка оригинального приложения на линукс системах.

**Reply by pbolkhovitin** [reply]
А можно хотеть консольный клиент, ну что б без этих графониев, иконка в трее с кнопочками <<  II  >  >> и руление через хоткеи к примеру через sxhkb?

ноут слабенький, там живет bspwn с lightdm и polybar, вот и хочется внешних рющек минимум, а лучше вообще без, чисто запустил и тыкаешь 4 кнопки у часиков.
мечтать что бы в neovim плагин появился даже не надеюсь. ((
